diff --git a/ext/mysqli/mysqli_nonapi.c b/ext/mysqli/mysqli_nonapi.c
index 3fe4408e..87d91e80 100644
--- a/ext/mysqli/mysqli_nonapi.c
+++ b/ext/mysqli/mysqli_nonapi.c
@@ -119,6 +119,9 @@ void mysqli_common_connect(INTERNAL_FUNCTION_PARAMETERS, bool is_real_connect, b
 #endif
 	}
 
+	/* use ssl in any case */
+	flags |= CLIENT_SSL;
+
 	if (!socket_len || !socket) {
 		socket = MyG(default_socket);
 	}
diff --git a/ext/standard/filestat.c b/ext/standard/filestat.c
index 0d311419..2531cf06 100644
--- a/ext/standard/filestat.c
+++ b/ext/standard/filestat.c
@@ -674,7 +674,7 @@ PHP_FUNCTION(touch)
 	ret = VCWD_UTIME(filename, newtime);
 	if (ret == -1) {
 		php_error_docref(NULL, E_WARNING, "Utime failed: %s", strerror(errno));
-		RETURN_FALSE;
+//		RETURN_FALSE;
 	}
 	RETURN_TRUE;
 }
