diff --git a/wp-admin/includes/upgrade.php b/wp-admin/includes/upgrade.php
index aee7beb..72c9599 100644
--- a/wp-admin/includes/upgrade.php
+++ b/wp-admin/includes/upgrade.php
@@ -278,19 +278,22 @@ if ( ! function_exists( 'wp_install_defaults' ) ) :
 To get started with moderating, editing, and deleting comments, please visit the Comments screen in the dashboard.
 Commenter avatars come from <a href="https://gravatar.com">Gravatar</a>.'
 		);
-		$wpdb->insert(
-			$wpdb->comments,
-			array(
-				'comment_post_ID'      => 1,
-				'comment_author'       => $first_comment_author,
-				'comment_author_email' => $first_comment_email,
-				'comment_author_url'   => $first_comment_url,
-				'comment_date'         => $now,
-				'comment_date_gmt'     => $now_gmt,
-				'comment_content'      => $first_comment,
-				'comment_type'         => 'comment',
-			)
-		);
+// quickfix for edb crashing when woocommerce uses alter table
+// see: https://github.com/edgelesssys/edgelessdb/issues/93
+//
+//		$wpdb->insert(
+//			$wpdb->comments,
+//			array(
+//				'comment_post_ID'      => 1,
+//				'comment_author'       => $first_comment_author,
+//				'comment_author_email' => $first_comment_email,
+//				'comment_author_url'   => $first_comment_url,
+//				'comment_date'         => $now,
+//				'comment_date_gmt'     => $now_gmt,
+//				'comment_content'      => $first_comment,
+//				'comment_type'         => 'comment',
+//			)
+//		);
 
 		// First page.
 		if ( is_multisite() ) {
